src/main/java/com/uber/jenkins/phabricator/RemoteFileFetcher.java

```diff
@@ -80,7 +81,9 @@ public String getRemoteFile() throws InterruptedException, IOException {
             maxLength = (int) source.length();
         }
         byte[] buffer = new byte[maxLength];
-        IOUtils.read(source.read(), buffer);
+        InputStream stream = source.read();
+        IOUtils.read(stream, buffer);
+        stream.close();
```

Comment:
Just a thought. You may want to close the stream in `finally` block and perform a basic check. Something like - 
```
try {
    IOUtils.read(stream, buffer);
} finally {
    try { if (stream != null) stream.close(); } catch(IOException e) { /* ignore */ }      
}
```

Reply:
Thanks for the recommendation. New to Java...